README.md

```diff
@@ -3,21 +3,28 @@
 # Installing preliminary Conda packages
 
 Some preliminary Conda packages can be installed as so. Replace `<CUDA
-version>` with either `9.2` or `10.0`.
+version>` with either `9.2`, `10.0`, or `10.1`. Also replace `<processor type>`
+with `cpu` or `gpu`.
 
 ```
-conda create -n ucx -c conda-forge -c jakirkham/label/ucx cudatoolkit=<CUDA version> ucx-proc=*=gpu ucx ucx-py python=3.7
+conda create -n ucx -c conda-forge -c jakirkham/label/ucx-new cudatoolkit=<CUDA version> ucx-proc=*=<processor type> ucx ucx-py python=3.7
 ```
 
 The ucx recipe can be found here: https://github.com/conda-forge/ucx-split-feedstock/tree/f13e882cc0566e795ff12f2a039f490ce1653698/recipe
 
+Note: These packages depend on the following system libraries being present:
+`libibverbs`, `librdmacm`, and `libnuma` (`numactl` on Enterprise Linux). So
+please install these with your Linux system's package manager.
```

Comment:
@randerzander, can you please take a look at this note and let me know your thoughts? 🙂

Reply:
I didn't have to install `libiverbs` or `librdmacm`, but my test was on a machine without IB.

Reply:
Interesting. I'm guessing they weren't already installed on your machine.

Yeah the package links to all 3, but it's possible that the UCX libraries that link to `libibverbs` and `librdmacm` are not loaded in your case. So you might not see an issue when they are not present.